src/main/java/ru/practicum/shareit/item/repository/InMemoryItemRepository.java

```diff
@@ -0,0 +1,55 @@
+package ru.practicum.shareit.item.repository;
+
+import org.springframework.stereotype.Repository;
+import ru.practicum.shareit.exceptions.NotFoundException;
+import ru.practicum.shareit.item.model.Item;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Repository
+public class InMemoryItemRepository implements ItemRepository {
+
+    private final Map<Long, Item> items = new HashMap<>();
+    private long lastId = 0;
+
+    public List<Item> getAllItems(Long userId) {
+        return items.values().stream().filter(item -> item.getOwnerId().equals(userId)).toList();
```

Comment:
Стримы нужно разбивать по строкам:
```suggestion
        return items.values().stream()
            .filter(item -> item.getOwnerId().equals(userId))
            .toList();
```